packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2756,8 +2786,23 @@ abstract class Element implements BuildContext {
   Set<InheritedElement> _dependencies;
   bool _hadUnsatisfiedDependencies = false;
 
+  void _debugCheckStateIsActive() {
+    assert(() {
+      if (_debugLifecycleState != _ElementLifecycle.active) {
+        throw new FlutterError(
+          'Looking up a deactivated widget\'s ancestor is unsafe.\n'
+          'At this point the state of the widget\'s element tree is no longer '
+          'stable.\nTo safely refer to a widget\'s ancestor in its dispose() method, '
```

Comment:
this text will change given the other comment above, but general rule for multiline strings:
- if you have an `\n`, end the string and start a new line, so that the code follows the string's line breaking.
- generally, paragraphs are fine here. Short paragraphs (lots of newlines) actually look a little weird because they make it look like we weirdly wrapped the text.